src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1505,6 +1507,39 @@ public boolean checkIfRequestForItemExistOrCreateAsynch(@NotNull final ItemStack
         return false;
     }
 
+    /**
+     * Check if a tag has been requested already or is in the inventory. If not in the inventory and not requested already, create request
+     *
+     * @param tag the requested tag.
+     * @return true if in the inventory, else false.
+     */
+    public boolean checkIfRequestForTagExistOrCreateAsynch(@NotNull final Tag<Item> tag, final int count)
```

Comment:
you should be able to simplify this one here now.

Reply:
it already is